src/styles/GlobalStyles.tsx

```diff
@@ -0,0 +1,21 @@
+// src/GlobalStyles.tsx
+import React from 'react';
+import { Global, css } from '@emotion/react';
+
+export function GlobalStyles() {
+  return (
+    <Global
+      styles={css`
+        /* 전역 테마 스타일: 폰트 및 기본 요소 */
+        html, body, #root {
+          height: 100%;
+          font-family: 'Helvetica Neue', sans-serif;
+        }
+        a {
+          text-decoration: none;
+          color: inherit;
+        }
+      `}
+    />
+  );
+}
```

Comment:
전역 스타일처럼 JSX를 반환하는 단순한 렌더링 컴포넌트일 경우, function보다는 const로 선언하는 게 권장됩니다! 또한 저희 모바일웹 뷰 사이즈와 rem 사용을 위한 폰트 사이즈도 적용해주시면 좋을 것 같아요!
```suggestion
const GlobalStyle = css`
  ${Reset}

  * {
    box-sizing: border-box;
  }
  
  html,
  body {
    display: flex;
    justify-content: center;
    min-height: 100vh;
    background-color: ${theme.colors.white};
    font-size: 62.5%;
  }

  #root { 
    width: 375px;
  }
`;
```